editions/server-external-js/tiddlywiki.info

```diff
@@ -2,8 +2,7 @@
 	"description": "Client-server edition with external tiddlywiki.js",
 	"plugins": [
 		"tiddlywiki/tiddlyweb",
-		"tiddlywiki/filesystem",
-		"tiddlywiki/highlight"
```

Comment:
Why has the highlight plugin been dropped here?

Reply:
IMO it's useless. ... The goal of the external-js edition is to make the HTML file as small as possible. The highlight.js plugin IMO is a copy/paste artefact from tw5.com-server and should have never been there.

Reply:
The original intent is to have seamless transition to/from standard server edition, so that the users would not see any discrepancies (in rendered tiddlers) between two editions.

Reply:
The size of the .html file created from the PR is 105kByte ... The size with the highlight plugin is trippled to 315kByte. 

This makes a significant difference if the file needs to be saved over the net to eg: github. Going from saving the file from 1 second to about 3 .... highly depending on the quality of the internet connection. 

If a user actively chooses to "burn" bandwidth, it's their choice. But IMO it's a "suboptimal experience" if it is forced upon users.